Added push-on-insert sample

adrianhall opened the pull request.

danderson00 commented:
Did you test?

From: Adrian Hall [mailto:notifications@github.com]
Sent: Friday, November 20, 2015 3:25 PM
To: Azure/azure-mobile-apps-node <azure-mobile-apps-node@noreply.github.com>
Subject: [azure-mobile-apps-node] Added push-on-insert sample (#220)

________________________________
You can view, comment on, or merge this pull request online at:

  https://github.com/Azure/azure-mobile-apps-node/pull/220

Commit Summary

  *   Updated package.json for samples to 2.0.0-alpha6
  *   Corrected URI in README.md
  *   Added push-on-insert sample
  *   Fixed promise spec

File Changes

  *   M samples/authentication/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-0> (2)
  *   M samples/authentication/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-1> (2)
  *   M samples/basic-app/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-2> (2)
  *   M samples/basic-app/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-3> (2)
  *   M samples/customApi/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-4> (2)
  *   M samples/dynamic-schema/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-5> (2)
  *   M samples/dynamic-schema/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-6> (2)
  *   M samples/personal-table/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-7> (2)
  *   M samples/personal-table/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-8> (2)
  *   A samples/push-on-insert/.gitignore<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-9> (37)
  *   A samples/push-on-insert/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-10> (17)
  *   A samples/push-on-insert/app.js<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-11> (27)
  *   A samples/push-on-insert/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-12> (15)
  *   A samples/push-on-insert/tables/TodoItem.js<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-13> (44)
  *   M samples/readonly-table/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-14> (2)
  *   M samples/readonly-table/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-15> (2)
  *   M samples/seeding-data/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-16> (2)
  *   M samples/seeding-data/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-17> (2)
  *   M samples/static-schema/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-18> (2)
  *   M samples/static-schema/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-19> (2)
  *   M samples/todo/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-20> (2)

Patch Links:

  *   https://github.com/Azure/azure-mobile-apps-node/pull/220.patch
  *   https://github.com/Azure/azure-mobile-apps-node/pull/220.diff

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-mobile-apps-node/pull/220>.

adrianhall commented:
Per our discussion – don’t merge just yet.  I’ll do an update when it works.

From: Dale Anderson [mailto:notifications@github.com]
Sent: Friday, November 20, 2015 3:40 PM
To: Azure/azure-mobile-apps-node <azure-mobile-apps-node@noreply.github.com>
Cc: Adrian Hall <adrian@shellmonger.com>
Subject: Re: [azure-mobile-apps-node] Added push-on-insert sample (#220)

Did you test?

From: Adrian Hall [mailto:notifications@github.com]
Sent: Friday, November 20, 2015 3:25 PM
To: Azure/azure-mobile-apps-node <azure-mobile-apps-node@noreply.github.com<mailto:azure-mobile-apps-node@noreply.github.com>>
Subject: [azure-mobile-apps-node] Added push-on-insert sample (#220)

________________________________
You can view, comment on, or merge this pull request online at:

https://github.com/Azure/azure-mobile-apps-node/pull/220

Commit Summary

* Updated package.json for samples to 2.0.0-alpha6
* Corrected URI in README.md
* Added push-on-insert sample
* Fixed promise spec

File Changes

* M samples/authentication/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-0> (2)
* M samples/authentication/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-1> (2)
* M samples/basic-app/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-2> (2)
* M samples/basic-app/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-3> (2)
* M samples/customApi/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-4> (2)
* M samples/dynamic-schema/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-5> (2)
* M samples/dynamic-schema/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-6> (2)
* M samples/personal-table/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-7> (2)
* M samples/personal-table/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-8> (2)
* A samples/push-on-insert/.gitignore<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-9> (37)
* A samples/push-on-insert/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-10> (17)
* A samples/push-on-insert/app.js<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-11> (27)
* A samples/push-on-insert/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-12> (15)
* A samples/push-on-insert/tables/TodoItem.js<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-13> (44)
* M samples/readonly-table/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-14> (2)
* M samples/readonly-table/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-15> (2)
* M samples/seeding-data/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-16> (2)
* M samples/seeding-data/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-17> (2)
* M samples/static-schema/README.md<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-18> (2)
* M samples/static-schema/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-19> (2)
* M samples/todo/package.json<https://github.com/Azure/azure-mobile-apps-node/pull/220/files#diff-20> (2)

Patch Links:

* https://github.com/Azure/azure-mobile-apps-node/pull/220.patch
* https://github.com/Azure/azure-mobile-apps-node/pull/220.diff

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-mobile-apps-node/pull/220>.


—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-mobile-apps-node/pull/220#issuecomment-158558586>.
